src/app/clusters/on-off-server/on-off-server.cpp

```diff
@@ -214,14 +214,15 @@ class DefaultOnOffSceneHandler : public scenes::DefaultSceneHandlerImpl
               ScenesManagement::ScenesServer::Instance().IsHandlerRegistered(endpoint, LevelControlServer::GetSceneHandler())))
 #endif
         {
+            VerifyOrReturnError(mTransitionTimeInterface.sceneEventControl(endpoint) != nullptr, CHIP_ERROR_INVALID_ARGUMENT);
             OnOffServer::Instance().scheduleTimerCallbackMs(mTransitionTimeInterface.sceneEventControl(endpoint), timeMs);
         }
 
         return CHIP_NO_ERROR;
     }
 
 private:
-    OnOffTransitionTimeInterface mTransitionTimeInterface = OnOffTransitionTimeInterface(Attributes::OnOff::Id, sceneOnOffCallback);
+    OnOffTransitionTimeInterface mTransitionTimeInterface = OnOffTransitionTimeInterface(OnOff::Id, sceneOnOffCallback);
```

Comment:
What will this Id be from? I believe we have both attributes and cluster as ID, it would be nice to be explicit.

Reply:
I would prefer Clusters:: for clarity, however that also seems nitpicky. Will defer to any other reviewer to see how subjective that is.

This is not the first time I have see bugs due to different IDs being used. One would have hoped we would be more type safe than that :/

Reply:
I see Stefan LGTMed it as-is, so I will close my comment.

Reply:
You could as well see in this file that cluster ID is used without Clusters:: everywhere.